CONTRIBUTING.md

```diff
@@ -30,6 +30,34 @@ works.
 
 This document will guide you through the contribution process.
 
+### Rules of thumb
+
+1. #### Provide motivation for the change
+   Why will this change make the code better; does it fix a bug, is it a new
+   feature, etc. This should be in the commit messages as well as in the PR
+   description.
```

Comment:
Do you mean having it there as well?
Since this is a general section, IMHO people should have this is mind when thinking about a contribution.